utils.py

```diff
@@ -0,0 +1,137 @@
+import ast
+import os
+import random
+import hashlib
+import hmac
+import pytz
+
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+from pytz import timezone
+
+slack_signing_secret = os.environ["SLACK_SIGNING_SECRET"]
+team_list_var = os.environ["TEAM_LIST"]
```

Comment:
Would probably do `CAPITAL_CASE` for these just because they should be essentially constants when the app starts.